### ISSUE TITLE
Accessibility: Editor Control Type Incorrect

### ISSUE THREAD

**Issue by lumills** [ckeditor/ckeditor4#4052]
## Type of report

Bug

## Provide detailed reproduction steps (if any)
1) Open a ckeditor 4 instance
2) Turn screen reader on (NVDA works)
3) Navigate into the Editor itself
4) Control type should be announced

### Expected result

Control type should be meaningful to the user, such as 'Editor'

### Actual result

Control type is not informational (I believe it is announced as 'group')

## Other details

* Browser: Firefox
* CKEditor version: 4

**Comment by f1ames** [comment]
Hi @lumills,

does it only concerns NVDA or any other screen reader too?

**Comment by lumills** [comment]
Hi @f1ames ,

Bug and repro that I'm seeing appears to be NVDA specifically

**Comment by Comandeer** [comment]
I can confirm that editor is presented in unhelpful way. It seems to be connected with a convoluted way in which we label the editor – see #2445, #2499 and #1904.

**Comment by lumills** [comment]
Hey @f1ames and @Comandeer just wanted to check if there is any sort of eta here / confirmation that it is in the pipeline? Thanks in advance

**Comment by f1ames** [comment]
Hello @lumills, unfortunately we have no ETA for this one. We have briefly analyzed this issue and related ones (see https://github.com/ckeditor/ckeditor4/issues/4052#issuecomment-632057850). This requires a solid rework on our side regarding how editor markup is created and so it's really hard to estimate how and when we can have a working solution.

Still, we are keeping an eye on it as accessibility is very important for us so there is a chance there will be a fix in a foreseeable future.

**Comment by naksenzh** [comment]
Hello @f1ames, we are tracking this issue as a blocker for our accessibility bugs. We would appreciate a lot if you could share the updates, please.

**Comment by f1ames** [comment]
Hello @naksenzh, as mentioned in https://github.com/ckeditor/ckeditor4/issues/4052#issuecomment-686342038, we have no clear ETA for this issue since it requires a significant amount of work in the editor itself. Any visible progress we will make on this issue (whatever positive or negative) will be posted here so you can track the status.

**Comment by naksenzh** [comment]
Hi @f1ames uny update or progress on this by chance ?

**Comment by f1ames** [comment]
Hello @naksenzh, unfortunately due to other priorities we were unable to push this issue forward for now.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5146
